### PR TITLE
ETFE-4592 Alignment make 'Density' units and design consistent

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
Just a small query

File: conf/messages.en [hunk 1]
```diff
@@ -352,7 +352,7 @@ itemDetails.key.netWeight = Net weight
 itemDetails.value.netWeight = {0} {1}
 itemDetails.key.density = Density
 itemDetails.value.density = {0}kg/m<sup>3</sup> {1}
-itemDetails.value.density.litres15 = (temperature of 15&deg;C)
+itemDetails.value.density.litres15 =
 itemDetails.value.density.litres20 = (temperature of 20&deg;C)
```
[the remark this answers is not in the document]
Comment (grahampaulcook) [reply conf/messages.en]: Pushed another commit to remove 20 degrees

File: conf/messages.en [hunk 2]
```diff
@@ -456,7 +456,7 @@ detailsSelectItem.value.alcoholicStrength = {0}%
 detailsSelectItem.key.density = Density
 detailsSelectItem.value.density = {0}kg/m<sup>3</sup> {1}
 detailsSelectItem.value.density.kilograms = kilograms
-detailsSelectItem.value.density.litres15 = (temperature of 15&deg;C)
+detailsSelectItem.value.density.litres15 =
 detailsSelectItem.value.density.litres20 = (temperature of 20&deg;C)
```
[the remark this answers is not in the document]
Comment (grahampaulcook) [reply conf/messages.en]: Pushed another commit to remove 20 degrees

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**